modules/hooks.nix

```diff
@@ -573,6 +573,13 @@ in
           entry = toString script;
           files = "\\.md$";
         };
+      mypy =
+        {
+          name = "mypy";
+          description = "Static type checker for Python";
+          entry = "${pkgs.mypy}/bin/mypy";
```

Comment:
We should add an option to set a custom binary path. `mypy` needs to be ran from an environment where all the Python packages with custom types are installed, i.e. your dev environment. See the `pylint.binPath` option for reference.

Reply:
Thanks, added a `binPath` option as advised.